build-system/dep-check-config.js

```diff
@@ -130,8 +133,29 @@ exports.rules = [
     mustNotDependOn: 'src/base-element.js',
   },
   {
-    filesMatching: 'extensions/**/*.js',
+    filesMatching: '**/*.js',
     mustNotDependOn: 'src/polyfills/**/*.js',
+    whitelist: [
+      // DO NOT add extensions/ files
+      '3p/polyfills.js->src/polyfills/math-sign.js',
+      '3p/polyfills.js->src/polyfills/object-assign.js',
+      'src/polyfills.js->src/polyfills/*.js',
+      'src/polyfills.js->src/polyfills/document-contains.js',
```

Comment:
Yeah, I think it is a feature.
We support wildcards on the left side of the `->` but not the right side.